builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -528,7 +533,7 @@ def validateDeploymentConfigurations(def prop) {
 		if (_runtime == "") {
 			error "Wrong configuration. Value for Key 'providerRuntime' is missing in the configuration"
 		} else {
-			def validRuntimes = ["nodejs4.3", "nodejs6.10", "python2.7", "java8"] //@TODO. Add more runtime supports.
+			def validRuntimes = ["nodejs4.3", "nodejs6.10", "python2.7", "java8", "go"] //@TODO. Add more runtime supports.
```

Comment:
doesn't it need some version?

Reply:
yes Go 1.x is the version now aws supports.